extras/accumulo-gis/fs/sbin/enable-iterators.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+ACCUMULO_USER="root"
+
+# Arg parsing
+if [ -z "${1}" ]; then
+  echo "No arguments provided - please specify either geowave or geomesa iterators as well as an appropriate accumulo namespace"
+  exit 1
+elif [ "${1}" = "geowave" ]; then
+  ITERATORS="geowave"
+elif [ "${1}" = "geomesa" ]; then
+  ITERATORS="geomesa"
+else
+  echo "Invalid iterator argument. Ensure that the first argument provided is either 'geowave' or 'geomesa'."
+  exit 1
+fi
+
+if [ -z "${2}" ]; then
+  echo "No namespace selected, using default ${ITERATORS} namespace, '${ITERATORS}'."
+  NAMESPACE="${ITERATORS}"
+else
+  NAMESPACE="${2}"
+fi
+
+# Ensure that the proper path structure is in place
+hadoop fs -mkdir /accumulo
+hadoop fs -mkdir /accumulo/classpath
+hadoop fs -mkdir /accumulo/classpath/${NAMESPACE}
+
+# GeoWave/Mesa specific operations
+if [ "${ITERATORS}" = "geomesa" ]; then
+  hadoop fs -put ${TMP_LIB_DIR}joda-time-2.3.jar \
+    /accumulo/classpath/${NAMESPACE}/joda-time-2.3.jar
```

Comment:
Can you double check that Accumulo classpath, I believe it's enough just to place it to [/accumulo-classpath/] (https://github.com/geotrellis/geodocker-cluster/blob/master/accumulo/fs/opt/accumulo/conf/accumulo-site.xml#L32)